charts/brigade/templates/apiserver/deployment.yaml

```diff
@@ -138,6 +138,23 @@ spec:
           value: {{ .Values.worker.image.pullPolicy }}
         - name: WORKSPACE_STORAGE_CLASS
           value: {{ .Values.worker.workspaceStorageClass }}
+        ports:
+        - name: healthz
+          containerPort: 8080
+        livenessProbe:
+          httpGet:
+            port: healthz
+            path: /healthz
+            scheme: HTTPS
+          initialDelaySeconds: 10
+          periodSeconds: 10
+        startupProbe:
+          httpGet:
+            port: healthz
+            path: /healthz
+            scheme: HTTPS
+          failureThreshold: 10
+          periodSeconds: 10
```

Comment:
Should we have a readiness probe as well?

Reply:
I wasn't previously familiar with startup probes. #TIL

Glancing at [the docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes) for my own edification, I noticed this:

> The trick is to set up a startup probe with the same command, HTTP or TCP check, with a failureThreshold * periodSeconds long enough to cover the worse case startup time.

So, I'm wondering if the failure threshold here should be higher than that of the liveness probe-- perhaps even substantially higher.

Reply:
Yes, will add a readinessProbe as well.

Re: StartupProbe; Sure...`30` sound good?

Reply:
At 10 second intervals-- that's a five minute grace period to come up. That sounds plenty forgiving to me.

👍